src/pages/steps/AdditionalInformationStep/FormScore.tsx

```diff
@@ -184,6 +137,52 @@ const getMinimumScore = (weights: Weights): number => {
   return minimumScore;
 };
 
+const BarometerWrapper = styled.div`
```

Comment:
You could just have used the `css` prop from styled components instead of making intermediate components

Reply:
I had missed you could use `css` on native elements, that's better thanks